# HG changeset patch
# User Rik <rik@octave.org>
# Date 1347989920 25200
#      Tue Sep 18 10:38:40 2012 -0700
# Node ID 2024c70329dfe045f4e81c48fcad570747eb1eb1
# Parent  c898dbe14e1d32687ad025d0396e45d421efcaad
.hgignore: Ignore gnulib/gnulib.mk file.

* .hgignore: Ignore gnulib/gnulib.mk file.

diff --git a/.hgignore b/.hgignore
--- a/.hgignore
+++ b/.hgignore
@@ -1,24 +1,24 @@
 syntax: regexp
 # The recurrent (^|/) idiom in the regexps below should be understood
 # to mean "at any directory" while the ^ idiom means "from the
 # project's top-level directory".
 
 # gnulib makes these silly backup files
-^.*~
+.~$
 
 # This directory gets created by gnulib during the build. 
 # It contains no Octave files.
 ^gnulib/
 
 # These directories mostly contain cruft during build time, but they
 # do contain some Octave code, so we gotta be a bit more careful about
 # what we ignore here
-^libgnu/.*\.([hc]$|sin$|valgrind$|charset$)
+^libgnu/.*\.([hc]$|sin$|valgrind$|charset$|mk$)
 ^m4/(?!ax_).+\.m4$
 
 # Emacs tools create these
 (^|/)TAGS$
 (^|/)semantic.cache$
 
 (/|^)Makefile\.in$
 ^INSTALL$
